orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.java

```diff
@@ -62,11 +62,16 @@ public ContextParameterProcessor(ContextFunctionConfiguration contextFunctionCon
   }
 
   public Map<String, Object> process(Map<String, Object> source, Map<String, Object> context, boolean allowUnknownKeys) {
+    ExpressionEvaluationSummary summary = new ExpressionEvaluationSummary();
+
+    return process(source, context, allowUnknownKeys, summary);
+  }
+
+  public Map<String, Object> process(Map<String, Object> source, Map<String, Object> context, boolean allowUnknownKeys, ExpressionEvaluationSummary summary) {
```

Comment:
Wouldn't mind seeing the arguments get wrapped so we're not going beyond 120.

Reply:
👍